x-pack/test_serverless/functional/test_suites/common/examples/search/warnings.ts

```diff
@@ -97,15 +98,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     afterEach(async () => {
-      await PageObjects.common.clearAllToasts();
+      await toasts.dismissAllToasts();
     });
 
     it('should show search warnings as toasts', async () => {
       await testSubjects.click('searchSourceWithOther');
 
       await retry.try(async () => {
-        const toasts = await find.allByCssSelector(toastsSelector);
-        expect(toasts.length).to.be(2);
+        const allToasts = await find.allByCssSelector(toastsSelector);
+        expect(allToasts.length).to.be(2);
```

Comment:
The `toastsSelector` in this file looks generic - might be a candidate to add an extra method to the toasts service, something like `getAllToasts` and/or `getToastCount`.